lib/ts_utils/paths.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from typing import Final
+
+TS_BASE_PATH: Final = Path(__file__).parent.parent.parent.relative_to(Path.cwd(), walk_up=True)
```

Comment:
`walk_up=True` is new in 3.12; it might be too early to require 3.12 here?

I think an alternative would be:
```suggestion
TS_BASE_PATH: Final = Path(__file__).parent.parent.parent.resolve().relative_to(Path.cwd())
```

Reply:
`resolve()` won't help as `Path(__file__)` is already absolute. I'll just skip the `walk_up` on Python < 3.12. This means, it's not possible to run the scripts from sub-directories, but that's the status quo anyway.